### PR TITLE
Add Acquring API guide

### PR DESCRIPTION
**Pull request by Magnus93**

### CODE REVIEW

File: docs/merchant/integration-guide/acquiring/api.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+An authorization often require a verification. 
+A verification is usually needed if the amount is large enough or when creating an initial recurring authorization.
+A verification is a Strong Customer Authentication (SCA) supported by the issuing bank. 
+
+The [Authorization API](../../authorization/create.html#create) let's you create authorizations for single time payments as well as recurring payments.
+After creating an authorization, the authorization can be [captured](../../authorization/captute.html), [canceled](../../authorization/cancel.html) or [refunded](../../authorization/refund.html).
+
+### Minimize PCI scope
+
+If you want to avoid handling card information to minimize PCI scope, you can use the [Card Input UI](../../card-input/embed.html#embeddable-component).
```
Comment (marcushanses) [inline docs/merchant/integration-guide/acquiring/api.md:18]: ... to minimize the PCI scope of your system, ...

File: docs/merchant/integration-guide/acquiring/api.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+The Card Input UI can perform 3D-Secure by using the Verification API to perform ([Read more here](../../card-input/verification.html#verification)).
```
Comment (marcushanses) [inline docs/merchant/integration-guide/acquiring/api.md:19]: to perform 3D Sercure?

File: docs/merchant/integration-guide/acquiring/api.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+It can also be [customized](../../card-input/style.html#styling) to better suit the style of a given website.
```
Comment (marcushanses) [inline docs/merchant/integration-guide/acquiring/api.md:20]: Intergiro Card Input can....

File: docs/merchant/integration-guide/acquiring/api.md (hunk 2 above)
Comment (marcushanses) [reply docs/merchant/integration-guide/acquiring/api.md:19]: Intergiro Card Input UI...

**Review by marcushanses — CHANGES_REQUESTED** [review]

**Review by robinnedergard — CHANGES_REQUESTED** [review]
Change datetime section in `docs/merchant/common/reference.md` to 
```
String formated as `"YYYY-MM-DDThh:mm:ss.fffK"`, e.g. `"2020-12-31T23:59:59.999Z"`.
```

**Review by nilssonemma — APPROVED** [review]

**Review by marcushanses — APPROVED** [review]

File: docs/merchant/customer/reference.md [hunk 4]
```diff
@@ -2,7 +2,8 @@
 
 ## Customer
 
-When creating a Customer, The “Customer Creatable” datatype is used. In the response from the endpoint an object of the “Customer data type will be returned.
+When creating a Customer, the “Customer Creatable” datatype is used. 
+In the response from the endpoint an object of the Customer data type will be returned.
```
Comment (robinnedergard) [inline docs/merchant/customer/reference.md:6]:
```suggestion
In the response from the endpoint, an object of the Customer data type will be returned.
```

File: docs/merchant/integration-guide/authentication.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+- private 
+- customer
+
+## Public API-keys
+
+Public API-keys are used to create new orders, new customers and new card tokens. 
+The public API-key is safe to use in the browser of your consumers.
+
+## Private API-keys
+Private API-keys are the most potent API-key and let’s you do everything available in the API.
```
Comment (robinnedergard) [inline docs/merchant/integration-guide/authentication.md:14]: No true in reality, sometimes you have to use customer or public key instead

File: docs/merchant/integration-guide/authentication.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+# Authentication
+
+Our API uses three different API-keys in forms of JWTs for authentication:
+- public
+- private 
+- customer
```
Comment (robinnedergard) [inline docs/merchant/integration-guide/authentication.md:6]: Will we mention agent audience to PSP:s?

File: docs/merchant/integration-guide/authentication.md (hunk 5 above)
Comment (robinnedergard) [inline docs/merchant/integration-guide/authentication.md:14]: Also inconstant use of space after head line

**Review by robinnedergard — CHANGES_REQUESTED** [review]

**Review by mayask — APPROVED** [review]

**Pull request merged**